### PR TITLE
Update ExprTk to tag 0.0.2

### PR DESCRIPTION
**Pull request by ArashPartow**

### CODE REVIEW

**Comment by ArashPartow** [comment]
@schuelermine when you have a moment can you please review and merge this PR.

**Comment by schuelermine** [comment]
I'm currently not at my PC so I can't build it, I will review soon!

**Review by schuelermine — CHANGES_REQUESTED** [review]
The hash was incorrect, it should be `sha256-ZV5nS6wEbKfzXhfXEtVlkwaEtxpTOYQaGlaxKx3FIvE=` (not just the format is different).
The commit message should be formatted as `exprtk: unstable-2021-12-31 -> unstable-2023-01-01`.

**Review by schuelermine — APPROVED** [review]

**Comment by schuelermine** [comment]
(I don't have permissions to merge)

**Comment by ArashPartow** [comment]
@wegank /  @NickCao  when you have a moment can you please review and merge this PR

File: pkgs/development/libraries/exprtk/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+  version = "unstable-2023-01-01";
 
   src = fetchFromGitHub {
     owner = "ArashPartow";
-    repo = "exprtk";
-    rev = "806c519c91fd08ba4fa19380dbf3f6e42de9e2d1";
-    hash = "sha256-5/k+y3gNJeggfwXmtAVqmaiV+BXX+WKtWwZWcQSrQDM=";
+    repo  = "exprtk";
+    rev   = "f46bffcd6966d38a09023fb37ba9335214c9b959";
+    hash  = "sha256-ZV5nS6wEbKfzXhfXEtVlkwaEtxpTOYQaGlaxKx3FIvE=";
```
Comment (wegank) [inline pkgs/development/libraries/exprtk/default.nix:11]:
```suggestion
    repo = "exprtk";
    rev = "f46bffcd6966d38a09023fb37ba9335214c9b959";
    hash = "sha256-ZV5nS6wEbKfzXhfXEtVlkwaEtxpTOYQaGlaxKx3FIvE=";
```

File: pkgs/development/libraries/exprtk/default.nix [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "exprtk";
-  version = "unstable-2021-12-31";
+  version = "unstable-2023-01-01";
```
Comment (wegank) [inline pkgs/development/libraries/exprtk/default.nix:5]:
```suggestion
  version = "0.0.2";
```

File: pkgs/development/libraries/exprtk/default.nix (hunk 1 above)
Comment (wegank) [inline pkgs/development/libraries/exprtk/default.nix:11]:
```suggestion
    repo = "exprtk";
    rev = version;
    hash = "sha256-ZV5nS6wEbKfzXhfXEtVlkwaEtxpTOYQaGlaxKx3FIvE=";
```

File: pkgs/development/libraries/exprtk/default.nix [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
```
Comment (wegank) [inline pkgs/development/libraries/exprtk/default.nix:3]:
```suggestion
stdenv.mkDerivation rec {
```

**Review by schuelermine — APPROVED** [review]

**Comment by wegank** [comment]
Also, please squash the commits, thanks!

**Review by schuelermine — CHANGES_REQUESTED** [review]
commit message should now end in "-> 0.0.2"

**Comment by schuelermine** [comment]
Sorry :/

**Comment by wegank** [comment]
@ofborg eval

**Pull request merged**